### PR TITLE
[Stable9.1] Backporting logging changes of PR #27144

### PR DESCRIPTION
**Pull request by noveens**
backporting logging changes of PR #27144

### CODE REVIEW

**Comment by noveens** [comment]
@PVince81 ,
Is this the right way to do backporting?

File: lib/private/BackgroundJob/Job.php [hunk 1]
```diff
@@ -49,7 +49,20 @@
 	public function execute($jobList, ILogger $logger = null) {
 		$jobList->setLastRun($this);
 		try {
+			//storing job start time
+			$jobStartTime = time();
+
+			\OCP\Util::writeLog('cron', 'Started background job of class : ' . get_class($this) . ' with arguments : ' . print_r($this->argument, true), \OCP\Util::DEBUG
+			);
```
Comment (jvillafanez) [inline lib/private/BackgroundJob/Job.php]: Move the `);` up or split the line better.

File: lib/private/BackgroundJob/Job.php [hunk 2]
```diff
@@ -49,7 +49,20 @@
...
+
 			$this->run($this->argument);
+
+			//storing job end time
+			$jobEndTime = time();
+			$timeTaken = $jobEndTime - $jobStartTime;
+
+			\OCP\Util::writeLog('cron', "Finished background job, the job took : $timeTaken seconds, " . "this job is an instance of class : " . get_class($this) . ' with arguments : ' . print_r($this->argument, true), \OCP\Util::DEBUG
+			);
```
Comment (jvillafanez) [inline lib/private/BackgroundJob/Job.php]: Move the ); up or split the line better.

**Comment by PVince81** [comment]
> Is this the right way to do backporting?

yes

**Comment by noveens** [comment]
@jvillafanez @PVince81 ,
updated the changes.

**Comment by jvillafanez** [comment]
👍

**Comment by noveens** [comment]
@PVince81 ,
merge this?

**Comment by jvillafanez** [comment]
Looks like we overlooked a missing whitespace...
```
{"reqId":"69jAKwn4pVcPHAaw32ds","remoteAddr":"82.159.139.58","app":"cron","message":"Finished background job, the job took : 0 seconds, this job is an instance of class : OC\\Command\\CommandJobwith arguments : O:33:\"OCA\\Files_Trashbin\\Command\\Expire\":1:{s:39:\"\u0000OCA\\Files_Trashbin\\Command\\Expire\u0000user\";s:5:\"admin\";}","level":0,"time":"2017-02-27T12:24:35+00:00","method":"GET","url":"\/cron.php","user":"--"}
```
Note the `Command\\CommandJobwith arguments`

@noveens could you create another PR to fix this?

**Comment by noveens** [comment]
@jvillafanez ,
PR on the way.

**Pull request merged**